template.yaml

```diff
@@ -549,6 +549,11 @@ Resources:
               "Type": "Task",
               "Resource": "arn:aws:lambda:AWS::REGION:AWS::ACCOUNT_ID:function:SungardAS-Onboarding-Update-Db",
               "InputPath": "$",
+              "Next": "TriggerCloudCheckr"
+            },
+            "TriggerCloudCheckr": {
+              "Type" : "Task",
+              "Resource": "arn:aws:lambda:AWS::REGION:AWS::ACCOUNT_ID:function:SungardAS-Onboarding-TriggerCloudChecker",
```

Comment:
Who will deploy this lambda function?

Reply:
This lambda function will be deployed as part of cloud checker stack which is being worked on by @mayankbharadwaj